### PR TITLE
gtksourceviewmm: init at 3.18.0

### PR DESCRIPTION
**Pull request by juliendehos**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by matthewbauer** [comment]
You should add in meta for: website, description, longDescription, and maintainers.

**Comment by juliendehos** [comment]
Done. This is just a C++ binding for gtksourceview so I didn't find any very interesting "longDescription".

File: pkgs/desktops/gnome-3/3.20/core/gtksourceviewmm/src.nix [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Autogenerated by maintainers/scripts/gnome.sh update
+
+fetchurl: {
+  name = "gtksourceviewmm-3.18.0";
```
Comment (matthewbauer) [inline pkgs/desktops/gnome-3/3.20/core/gtksourceviewmm/src.nix]: Is there a 3.20 version available?

Comment (juliendehos) [inline pkgs/desktops/gnome-3/3.20/core/gtksourceviewmm/src.nix]: No. There is a  3.21.2 version but it's a development version (https://developer.gnome.org/gtksourceviewmm/).

**Pull request merged**